src/dialog/dialog.scss

```diff
@@ -1,111 +1,19 @@
-.modal-hide {
-  display: none !important;
+.Dialog {
```

Comment:
@vdepizzol should we try and manage all component specific cases as props within `overlay.scss` or allow component-specific CSS on top?

Reply:
@langermank I think it depends on the component-specific CSS on top 😇. I'd expect any piece that is reusable to live in the `overlay` pattern.